src/Search/Console.php

```diff
@@ -343,26 +342,15 @@ public function rdsReindexCommand(InputInterface $input, OutputInterface $output
         $this->logger->info('RDS articles added to indexing queue.');
     }
 
-    public function reviewedPreprintReindexCommand(InputInterface $input, OutputInterface $output)
-    {
-        $this->logger->info('Reindex reviewed preprints...');
-        $ids = [];
-        foreach (array_keys($this->config['reviewed_preprints']) as $id) {
-            $this->logger->info("Queuing reviewed preprint article $id");
-            $this->enqueue('article', $id);
-            $this->logger->info("Queuing reviewed preprint $id");
-            $this->enqueue('reviewed-preprint', $id);
-            $ids[] = $id;
-        }
-        $output->writeln('Queued: '.implode(', ', $ids).' (reviewed-preprint and article)');
-        $this->logger->info('Reviewed preprints added to indexing queue.');
-    }
-
     public function reviewedPreprintPurgeCommand(InputInterface $input, OutputInterface $output)
```

Comment:
We could create a follow up ticket to create a single command to allow purging by type.

Reply:
Ticket has been created: https://github.com/elifesciences/issues/issues/7985